apps/main/src/llamalend/features/manage-soft-liquidation/ui/tabs/ClosePosition.tsx

```diff
@@ -64,14 +67,14 @@ export const ClosePosition = ({ debtToken, collateralToRecover, canClose, status
     </Stack>
 
     <AlertClosePosition />
-    {+canClose.missing > 0 && debtToken?.symbol && (
-      <AlertAdditionalCrvUsd debtTokenSymbol={debtToken?.symbol} missing={+canClose.missing} />
+    {missing && +missing > 0 && debtToken?.symbol && (
+      <AlertAdditionalCrvUsd debtTokenSymbol={debtToken?.symbol} missing={+missing} />
     )}
 
     <Stack gap={Spacing.xs}>
       <Button
-        disabled={status === 'close' || +canClose.missing > 0 || +(debtToken?.amount ?? '') <= 0}
-        onClick={() => onClose(debtToken, collateralToRecover)}
+        disabled={status === 'close' || (missing && +missing > 0) || +(debtToken?.amount ?? '') <= 0}
```

Comment:
We check twice the same thing in two ways, one of them relying in NaN behavior.
Some consistency and clarity:
```suggestion
        disabled={status === 'close' || +(missing ?? '0') > 0 || +(debtToken?.amount ?? '0') <= 0}
```